glance_plugin/tests/resources/test-image-start.yaml

```diff
@@ -0,0 +1,31 @@
+
+tosca_definitions_version: cloudify_dsl_1_3
+
+imports:
+  - http://www.getcloudify.org/spec/cloudify/3.4/types.yaml
+  - plugin.yaml
+
+inputs:
+  use_password:
+    type: boolean
+    default: false
+
+node_templates:
+  image:
+    type: cloudify.openstack.nodes.Image
+    properties:
+      image_url: test_url
```

Comment:
Image_url should be outside - at least I couldn't find in glance docs an attribute to image.create() that would work with url. I'll remove image_url from this test.